### PR TITLE
[EZ] Add logs for some basic training params so that we can verify in…

### PR DESCRIPTION
**Pull request by fduwjj**
As title, while testing on 405B model, I found that we need to somehow need the logs for some training params. So added some here. Tested locally and the logging is shown as in the screenshot:

image

### CODE REVIEW

File: train.py [hunk 1]
```diff
@@ -232,6 +232,12 @@ def loss_fn(pred, labels):
     model_config.max_seq_len = job_config.training.seq_len
 
     logger.info(f"Building {model_name} {job_config.model.flavor} with {model_config}")
+    logger.info(
+        f"Detailed training config, local_batch_size: {job_config.training.batch_size}, "
+        f"seq_len: {job_config.training.seq_len}, "
+        f"total_steps: {job_config.training.steps}({job_config.training.warmup_steps}), "
+        f"activation_checkpoint: {job_config.activation_checkpoint.mode}"
```
Comment (tianyu-l) [inline train.py]: we can remove AC here, as it has been logged at https://github.com/pytorch/torchtitan/blob/main/torchtitan/parallelisms/parallelize_llama.py#L449

File: train.py [hunk 2]
```diff
@@ -232,6 +232,12 @@ def loss_fn(pred, labels):
     model_config.max_seq_len = job_config.training.seq_len
 
     logger.info(f"Building {model_name} {job_config.model.flavor} with {model_config}")
+    logger.info(
```
Comment (tianyu-l) [inline train.py]: let's move this logging to https://github.com/pytorch/torchtitan/blob/main/train.py#L358

File: train.py [hunk 3]
```diff
@@ -232,6 +232,12 @@ def loss_fn(pred, labels):
...
+        f"Detailed training config, local_batch_size: {job_config.training.batch_size}, "
```
Comment (tianyu-l) [inline train.py]: let's remove underscore as they do not always refer to variable name, e.g. use `local batch size`, `sequence length`, `total steps`

**Review by tianyu-l — APPROVED** [review]
thanks for the improvement! pls address comments before merge.

File: train.py (hunk 1 above)
Comment (fduwjj) [reply train.py]: hmmm, the logging info is pretty scattered... Ideally we want users to have one place of every config..

**Pull request merged**